classes/helper.php

```diff
@@ -17,6 +17,7 @@
 namespace tool_excimer;
 
 use core_filetypes;
+use moodle_url;
```

Comment:
nitpick: I generally prefer to just \ this inline, purely because its a little cleaner IMO.